### PR TITLE
[FT-3014] Integrate Github actions

### PR DESCRIPTION
**Pull request by sol-214**

### CODE REVIEW

**Comment by sol-214** [comment]
Need to add `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to Secrets.

**Comment by sol-214** [comment]
...and remove Codeship as a required check.

File: .github/workflows/aws.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: eu-west-1
+      CLOUDFRONT_DISTRITBUTION_ID: E2FIFY2PI64GVA
+      S3_BUCKET_URI: s3://fantech.assets/fantasysport/hosted
+      AUTH_URL: https://auth.fantech.io
+      USE_SSL: false
+      DEFAULT_AUTH_PROVIDER: billfold-wallet
+      API_KEY: d80eb31e4e54c04de949f5c56049760c
```
Comment (mheki) [inline .github/workflows/aws.yml]: have we always had production api key in github? We could use secrets...

Comment (sol-214) [reply .github/workflows/aws.yml]: I duno why it's there tbh, it's not used, but it's public:
https://github.com/rushpartners/fantasy-js-sdk/blob/develop/.env#L4

File: .github/workflows/aws.yml [hunk 2]
```diff
@@ -3,12 +3,8 @@ on:
   push:
     branches:
       - develop
-      - master
-      - hotfix/*
-  pull_request:
-    types: [opened, synchronize]
-    branches-ignore:
-      - master
+  release:
```
Comment (vikkio88) [inline .github/workflows/aws.yml]: I wouldnt use releases for this as it is not going to be released

File: .github/workflows/aws.yml (hunk 1 above)
Comment (mheki) [reply .github/workflows/aws.yml]: fair enough

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Project: FT JS-SDK
 
-[![wercker status](https://app.wercker.com/status/a47724f8bd81bbe45a29ee1576a89642/m/ "wercker status")](https://app.wercker.com/project/byKey/a47724f8bd81bbe45a29ee1576a89642)
+![Test](https://github.com/rushpartners/fantasy-js-sdk/workflows/Test/badge.svg)
```
Comment (mheki) [inline README.md]: you could also add `deploy to production`, `deploy to staging` badges 👍

**Pull request merged**